Resources/Prototypes/silicon-laws.yml

```diff
@@ -499,7 +499,7 @@
   - Nutimov5
   obeysTo: laws-owner-crew
 
-# ion storm random lawsets
+ # ion storm random lawsets
```

Comment:
Unnecessary change to upstream file. Extra space before comment

Reply:
Resolved.